proto/ligato/vpp/nat/nat.proto

```diff
@@ -178,3 +178,22 @@ message VirtualReassembly {
     // If set to true fragments are dropped, translated otherwise.
     bool drop_fragments = 4;
 }
+// Nat44EdVrfTable defines a vrf table for NAT44.
+message Nat44EdVrfTable {
```

Comment:
It's working only with Endpoint dependent NAT44. That's the reason why I didn't generalize the naming to Nat44VrfTable.

Reply:
Thing is that we don't currently make distinction in our Proto API between in endpoint dependent vs. independent, beside the one boolean in global config. I think you could name the message `Nat44VrfTable` and provide better comment that includes info that it is used for ED NAT only. Additionally to this, you could add warning in log informing that the VRF table is only used if ED NAT is enabled.

Looking a bit out of the box, it might be that our Proto API might need a bit of refactor (which actually might already be in our backlog).. I am wondering if both ED and EI NAT can be enabled at the same time. In that case we definitely need some kind of refactor for the API to allow this.

Reply:
Agreed, I think we should just call it `Nat44VrfTable` (+ explanatory comments) to keep it in line with current conventions. In the future refactoring of NAT seems likely, as it looks like to me that ED NAT and EI NAT features are diverging in VPP as time goes on. But without thorough investigation of these NAT modes in VPP (similarities/differences, use cases for each) it is hard to make the call on how should we approach the refactor - Split it in the Proto API? Or just creating 2 separate NAT VPP handlers would be enough? etc. Anyway, I am not knowledgeable enough in this area to have any strong opinions...